warehouse/pyproject.toml

```diff
@@ -21,13 +21,13 @@ python-slugify = "^6.1.2"
 sentry-sdk = "^1.9.8"
 backoff = "^2.2.1"
 gcsfs = "^2023.1.0"
-dbt-metabase = "^0.9.14"
+dbt-metabase = "0.10.3"
```

Comment:
Are future versions incompatible which is why this is pinned?

Reply:
The `dbt-metabase` package recently moved to a 1.0, which I'm going to create a new issue to move to - it includes breaking changes for several arguments, and so it becomes a slightly larger project to switch it out.

I pinned to a specific version here as a sort of implicit flag that 0.10.3 appears to be the end of the line for pre-1.0 versions of the package, but that's better addressed via a note, so I'm putting in another commit shortly to float the minor version once again and document that we don't anticipate further pre-1.0 updates to the package.